source/docs/prismic/integration-fields.md

```diff
@@ -0,0 +1,289 @@
+
+This allows to design rich custom content types that will mix and match data from several datasources **without introducing complexity for your frontend.**
+
+## Register Integration fields in Front-Commerce
+
+Registering a new Integration field allows to expose custom data in an API that can be used by Prismic. Front-Commerce's Prismic module provides a generic way to register Integration fields.
+
+See the [Implementations](#Implementations) section to learn more about existing implementations.
+
+<blockquote class="note">
+**Note:** In this example, we will use the [`SitemapableIntegrationField`](#SitemapableIntegrationField) implementation for more concise examples. Keep in mind that any `IntegrationField` implementation can be registered in a similar way.
+</blockquote>
+
+To register a new Integration field, you must use the `loaders.Prismic.registerIntegrationField()` method of the Prismic loader. You can then provide a name that will identify it across Front-Commerce and an implementation.
+
+It can be done from [a GraphQL module `contextEnhancer`](/docs/reference/graphql-module-definition.html#contextEnhancer-optional) as in the example below:
+
+```js
+export default {
+  // […]
+  dependencies: [
+    "Prismic/Core",
+    "Magento2/Catalog/Categories",
+    "Magento2/Catalog/Products"
+  ],
+  contextEnhancer: ({ loaders }) => {
+    const { SitemapableIntegrationField } = loaders.Prismic.integrationFields;
+
+    loaders.Prismic.registerIntegrationField(
+      "Category", // <- integration field identifier
+      new SitemapableIntegrationField("Category", loaders.Sitemap, ({ id }) => {
+        return loaders.Category.load(id)
+      })
+    );
+    loaders.Prismic.registerIntegrationField(
+      "Product",
+      new SitemapableIntegrationField("Product", loaders.Sitemap, ({ id }) => {
+        // The SitemapableIntegrationField tries to guess what the id is (in this case, the SKU is used)
+        return loaders.Product.load(id)
+      })
+    );
+
+    return {};
+  },
+};
+```
+
+When Integration fields have been registered, Front-Commerce will expose an API endpoint for each of them at the `/prismic/integration/{name}` URL. Examples:
+- http://localhost:4000/prismic/integration/Category
+- http://localhost:4000/prismic/integration/Product
+
+This API is protected with a security token defined in the `FRONT_COMMERCE_PRISMIC_WEBHOOK_SECRET` environment variable. To test it, you can run the following `curl` commands:
+```
+curl --user 'a-secret-defined-in-webhook-configuration:' http://localhost:4000/prismic/integration/Product
+curl --user 'a-secret-defined-in-webhook-configuration:' http://localhost:4000/prismic/integration/Product?page=2
+```
+
+You must see a paginated JSON description of your data matching [Prismic's Custom API Format](https://prismic.io/docs/core-concepts/integration-fields-setup#pull-data-into-prismic-from-a-custom-api). **If this is not happening, you might have an error in the registration.**
```

Comment:
is there a debug flag in that area ? if so might be worth mentioning it to help troubleshooting